src/MOI_wrapper.jl

```diff
@@ -92,25 +91,62 @@ Base.@kwdef struct ConicCache
     b::Vector{Float64}
     c::Vector{Float64}
 end
-
 const CACHE_TYPE = Union{
     Nothing,
     QPCache,
     ConicCache,
 }
 
+Base.@kwdef struct QPForwCache
+    dz::Vector{Float64}
+    dλ::Vector{Float64}
+    dν::Vector{Float64}
+end
+Base.@kwdef struct QPBackCache
```

Comment:
Do we need to define two different types for forward and backward in the QP case?

Reply:
- [ ] It is a little drafty, I am was planning to dispatch on them.
Bu now there are separate fields so I might just merge them.

Reply:
merged these two